packages/wishes_api/lib/src/models/wish.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+import 'package:meta/meta.dart';
+import 'package:uuid/uuid.dart';
+import 'package:wishes_api/wishes_api.dart';
+
+part 'wish.g.dart';
+
+/// {@template wish}
+/// A single wish item.
+///
+/// Contains a [title], [description] and [id], in addition to a [isFulfilled]
+/// flag.
+///
+/// If an [id] is provided, it cannot be empty. If no [id] is provided, one
+/// will be generated.
+///
+/// [Wish]s are immutable and can be copied using [copyWith], in addition to
+/// being serialized and deserialized using [toJson] and [fromJson]
+/// respectively.
+/// {@endtemplate}
+@immutable
+@JsonSerializable()
+class Wish extends Equatable {
+  /// {@macro wish}
+  Wish({
+    String? id,
+    required this.title,
+    this.description = '',
+    this.isFulfilled = false,
+  })  : assert(
+          id == null || id.isNotEmpty,
+          'id can not be null and should be empty',
```

Comment:
```suggestion
          'id cannot be null and should be empty',
```

Reply:
Seems that has to be changed on the tutorial as well then